test/routes/healthz.test.js

```diff
@@ -0,0 +1,27 @@
+describe('Healthz test', () => {
+  let createServer
+  let server
+
+  beforeAll(async () => {
+    createServer = require('../../server')
+  })
+
+  beforeEach(async () => {
+    server = await createServer()
+    await server.initialize()
+  })
+
+  test('GET /liveness route returns 200', async () => {
```

Comment:
The route is /healthz not /liveness, not functionally incorrect, but a bit misleading.

Reply:
Good spot.  Fixed.